src/modals/UnlockChunks/UnlockStatus.tsx

```diff
@@ -0,0 +1,42 @@
+import { formatDuration, intervalToDuration } from 'date-fns';
+import { useApi } from '../../contexts/Api';
+import { useSessionEra } from '../../contexts/SessionEra';
+
+type Props = {
```

Comment:
A little divergence from the convention I noticed in this repo, but this is for good reasons:
1. `interface`s, which have been used in this repo for props so far, are dangerous because of their feature that if there are multiple with the same name in a scope, they get merged - a very simple way to find oneself in a very hard to resolve bug situation. `type`s, on the other hand, work as you'd expect them to work.
2. Props kept close to their component is a more efficient technique than a having a separate _bucket file_ of props types (separation of concerns vs separation of technologies). Props are an inseparable part of a component and only make sense in context of their component, so having them in a separate file makes it harder to read the code.

Reply:
Convinced.